assets/js/player.js

```diff
@@ -41,6 +41,12 @@ embed_url = location.origin + '/embed/' + video_data.id + embed_url.search;
 var save_player_pos_key = "save_player_pos";
 
 videojs.Vhs.xhr.beforeRequest = function(options) {
+    if (options.headers) {
+        if (options.headers.Range) {
+            options.uri = options.uri + '&range=' + options.headers.Range.split("=")[1];
+            delete options.headers.Range;
```

Comment:
Do note that removing the `Range` header needs to be handled on the server side.
@unixfox it works for you because you don't use Invidious' proxy!

Reply:
> Do note that removing the `Range` header needs to be handled on the server side. @unixfox it works for you because you don't use Invidious' proxy!

It's mostly handled on the youtube server side but I see that there are no code related to the `Range` parameter inside invidious but probably not mandatory to work because it worked fine on a vanilla invidious installation.

Reply:
Part of the code removed.

Reply:
> It's mostly handled on the youtube server side but I see that there are some code related to the `Range` parameter inside invidious

Tbh, the proxying part should be reworked, but last time I've tried to touch that part of the code, everything broke (see #2900).

Reply:
That's true. The range header being transformed into a query string is really only useful for me because I run a big instance with multiple external proxy servers. Now that I think again about it, I don't think it makes sense to modify that for everyone.